### ISSUE TITLE
Add Solution in Swift Language

### ISSUE THREAD

**Issue by sagarchoudhary96** [sagarchoudhary96/30-Days-Of-Code#49]
Solutions are welcomed for each problem in Swift language. Please check **contributing Guidelines** before starting.  You can register for **Hacktoberfest** [here](https://hacktoberfest.digitalocean.com/) to receive **free swags** once you raise 4 PR.

- [ ] Day 0 Assigned to @penelopearaujo
- [ ] Day 1
- [ ] Day 2 
- [ ] Day 3
- [ ] Day 4
- [ ] Day 5
- [ ] Day 6
- [ ] Day 7 
- [ ] Day 8
- [ ] Day 9  
- [ ] Day 10 
- [ ] Day 11
- [ ] Day 12
- [ ] Day 13
- [ ] Day 14
- [ ] Day 15
- [ ] Day 16
- [ ] Day 17
- [ ] Day 18
- [ ] Day 19
- [ ] Day 20
- [ ] Day 21
- [ ] Day 22
- [ ] Day 23
- [ ] Day 24
- [ ] Day 25
- [ ] Day 26
- [ ] Day 27
- [ ] Day 28
- [ ] Day 29

**Comment by kristoft** [comment]
I'd like to add solution for Day 1 & Day 2 :)

**Comment by penelopearaujo** [comment]
Right! I can do the next three - days 3, 4 and 5

**Comment by rogerthat39** [comment]
I can work on days 6 and 7

**Comment by sagarchoudhary96** [comment]
@penelopearaujo @rogerthat39 @kristoft , assigned to you 👍

**Comment by rogerthat39** [comment]
I'd like to continue working on this, could I work on days 8, 9 and 10?

**Comment by sagarchoudhary96** [comment]
sure @rogerthat39 go ahead 👍

**Comment by ham118** [comment]
I'd like to add the solutions for Day11 to Day14.

**Comment by sagarchoudhary96** [comment]
@ham118 go ahead 👍

**Comment by Kuljeet-123** [comment]
@sagarchoudhary96 I'd like to provide solution for Day16 .

**Comment by sagarchoudhary96** [comment]
@Kuljeet-123 sure go ahead will assign to you.

**Comment by Kuljeet-123** [comment]
@sagarchoudhary96 please review my PR and merge into your repo and my Pr id is #80 .

**Comment by sagarchoudhary96** [comment]
Done @Kuljeet-123  looking forward for more contributions.

**Comment by sagarchoudhary96** [comment]
@kristoft @penelopearaujo @ham118 @rogerthat39, we are halfway completing this task. would be great
if you guys would like to take up more questions to solve.

**Comment by rogerthat39** [comment]
I could work on days 17-19?

**Comment by sagarchoudhary96** [comment]
@rogerthat39 sure go ahead.

**Comment by sagarchoudhary96** [comment]
@rogerthat39  any updates, is any one of 17-19 is completed? you can raise PR for individual solutions too.

**Comment by rogerthat39** [comment]
@sagarchoudhary96 17 and 18 are now completed, but 19 is not available in Swift. If you like, I could do Day 20 instead?

**Comment by sagarchoudhary96** [comment]
@rogerthat39 sure go ahead

**Comment by sagarchoudhary96** [comment]
@kristoft @penelopearaujo @ham118 @rogerthat39 @Kuljeet-123  we are about to finish this.  would you guys like to take up more questions from available on?

**Comment by Kuljeet-123** [comment]
@sagarchoudhary96  I would like to take Day 21 ..please assign me this.

**Comment by saurabh-7777** [comment]
@sagarchoudhary96  I would like to take Day 22 ..please assign me this.

**Comment by sagarchoudhary96** [comment]
@Kuljeet-123 @saurabh-7777  go ahead :)

**Comment by saurabh-7777** [comment]
@sagarchoudhary96  please review my PR and merge into your repo and my PR id is #92 .

**Comment by Kuljeet-123** [comment]
@sagarchoudhary96  please I have added Day 21 solution please review it and merge into your repo and my PR id is #93 .

**Comment by ham118** [comment]
@sagarchoudhary96 I'd like to add a solution for Day23.

**Comment by sagarchoudhary96** [comment]
@ham118 go ahead👍

**Comment by ham118** [comment]
@sagarchoudhary96 Please review PR #96

**Comment by NicolasCombe5555** [comment]
@sagarchoudhary96 There is no option for a swift compiler on Day 27. Can i take Day 26 challenge "Nested Logic" ?

**Comment by sagarchoudhary96** [comment]
Yeah sure go ahead

**Comment by NicolasCombe5555** [comment]
@sagarchoudhary96 thank you, its done. If you can check my pr and test it on hackerrank.

**Comment by sagarchoudhary96** [comment]
Great. Sure will check in a while :)

**Comment by NicolasCombe5555** [comment]
@sagarchoudhary96 i also added day 28.
